Cart/cart.css

```diff
@@ -0,0 +1,180 @@
+main {
+    width: 100%;
+    height: calc(100vh - 76px);
+    background-color: var(--secondary-background);
+}
+
+.page-title {
+    text-align: center;
+    padding: 2rem;
+}
+
+.cart-container {
+    width: 70%;
+    margin: 0 auto;
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 2fr 1fr;
+    grid-template-areas: "cart-products cart-summary";
+}
+
+.cart-products {
+    width: 100%;
+    grid-area: cart-products;
+}
+
+.cart-summary {
+    width: 100%;
+    grid-area: cart-summary;
+}
+
+/* landscape product card css */
+
+.horizontal-card {
+    width: auto;
+}
+
+.horizontal-card .horizontal-card-info {
+    padding: 2rem;
+    flex-shrink: 1;
+    align-items: flex-start;
+    justify-content: flex-start;
+}
+
+.horizontal-card .card-img {
+    max-width: 40%;
```

Comment:
this can be replaced by width: 40%

Reply:
did the suggested change. thank you @raieegulhane